### PR TITLE
add-input-field-with-electricity-costs

### PR DESCRIPTION
**Pull request by sirtobiwan**

### CODE REVIEW

File: pages/savings.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import Button from "@/components/Button";
+import CardSavings from "@/components/CardSavings";
 import styled from "styled-components";
 
 export default function Savings() {
   return (
     <>
-      <p>Savings:</p>
+      <CardSavings></CardSavings>
```
Comment (onemanwenttomow) [inline pages/savings.js:8]: I think that if this component does not have any `children` then we can make it self closing. i.e. 

``` js
<CardSavings />
```

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice work! I added a few thoughts.

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice work! I added a few thoughts.

File: components/CardSavings/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+  
+  return (
+    <StyledCard>
+      <StyledContent>
+        <StyledLabel>Enter your ElectricityCosts:</StyledLabel>
+        <StyledData><ElectricityCosts/></StyledData>
+      </StyledContent>
+      <StyledContent>
```
[the remark this answers is not in the document]
Comment (sirtobiwan) [reply components/CardSavings/index.js:103]: yes these will be input and label

**Review by onemanwenttomow — APPROVED** [review]
LGTM 🎉

**Pull request merged**